src/swap/transfer.ts

```diff
@@ -44,29 +43,22 @@ export function makeTransferPlugin(
         publicAddress: toAddress
       } = await request.toWallet.getReceiveAddress()
 
-      const tx: EdgeTransaction = await request.fromWallet.makeSpend({
+      const spendInfo = {
         currencyCode: request.fromCurrencyCode,
         spendTargets: [
           {
             nativeAmount: request.nativeAmount,
             publicAddress: toAddress
           }
         ]
-      })
+      }
```

Comment:
In the previous version of this PR, you added a bunch of swap parameters to this `spendInfo`. Those changes no longer seem to exist, so we need to get them back.

Reply:
Found it:

```ts
          swapData: {
            isEstimate: false,
            plugin: { ...swapInfo },
            payoutAddress: toAddress,
            payoutCurrencyCode: requestInner.fromCurrencyCode,
            payoutNativeAmount: requestInner.nativeAmount, // Wrong
            payoutWalletId: requestInner.toWallet.id
          }
```

Reply:
yup missed that, done